x-pack/plugins/maps/public/classes/sources/es_geo_grid_source/es_geo_grid_source.tsx

```diff
@@ -303,55 +354,72 @@ export class ESGeoGridSource extends AbstractESAggSource {
       }),
     });
 
-    return convertRegularRespToGeoJson(esResponse, this._descriptor.requestType);
+    return convertRegularRespToGeoJson(esResponse, this._descriptor.requestType || RENDER_AS.POINT);
```

Comment:
No need for this check since requestType is set in constructor. Just use null operator